.github/workflows/ci.yml

```diff
@@ -85,7 +85,7 @@ jobs:
         run: npm run test
 
       - name: ⬆️ Upload coverage report
-        uses: codecov/codecov-action@v4
+        uses: codecov/codecov-action@v3
```

Comment:
Is there a reason why we're downgrading here?

Reply:
It looks like v4 has been deleted.
https://github.com/codecov/codecov-action/issues/1089